examples/02-javascript-basics/bob.js

```diff
@@ -6,15 +6,38 @@ He retorts 'Calm down, I know what I'm doing!' if you yell a question at him.
 He says 'Fine. Be that way!' if you address him without actually saying anything.
 He answers 'Whatever.' to anything else.
 */
+function isUpperCaseOrNumeric(x) {
```

Comment:
@anneilustre good use of a separate function.

Reply:
@anneilustre Although, giving the name `isUpperCaseOrNumeric` implies that it returns only boolean values, however it returns an integer as well. I think we should change the name into something more descriptive.